oap-server/server-cluster-plugin/cluster-etcd-plugin/src/main/java/org/apache/skywalking/oap/server/cluster/plugin/etcd/EtcdCoordinator.java

```diff
@@ -51,6 +59,9 @@
 
     private static final Integer KEY_TTL = 45;
 
+    @Setter
+    private HealthCheckMetrics healthChecker;
```

Comment:
As this is required in the process, please don't use the `setter`. Initial this in the constructor.

Reply:
done